kolibri/core/assets/src/vue/session-nav-widget/index.vue

```diff
@@ -28,7 +28,7 @@
               <p id="dropdown-usertype">{{ userkind }}</p>
             </li>
             <li id="logout-tab">
-              <div tabindex="0" @keyup.enter="userLogout" @click="logout" :aria-label="logOutText">
+              <div tabindex="0" @keyup.enter="userLogout" @click="logout" :aria-label="$tr('logOut')">
```

Comment:
This does not technically render as a button (otherwise it would not require `tabindex="0"`), and we might want to revisit it at some point. ATM it might be redundant, but it is kosher in sense that it neither throws any validation errors nor impedes proper usage... :wink: